src/shell/pmi/pmi.c

```diff
@@ -104,7 +104,7 @@ static void shell_pmi_abort (void *arg,
      */
     flux_shell_raise ("exec",
                       0,
-                      "MPI_Abort%s%s",
```

Comment:
Commit message typo: "when the broker call" should be "when the broker calls". 

Also
> the shell PMI plugin logs an exception message calls out MPI_Abort()

"that calls out MPI_Abort()?"

Reply:
Thanks! I pushed that commit message fix and will set MWP.